src/Tests/Execute/InferenceRunner.test.ts

```diff
@@ -88,16 +93,18 @@ suite('Execute', function() {
 
     suite('#constructor()', function() {
       test('is constructed with params', function() {
-        let runner = new InferenceRunner(backend, modelPath, inputSpec);
+        let runner = new InferenceRunner(backend, executor, modelPath, inputSpec);
         assert.strictEqual(runner.backend, backend);
+        assert.strictEqual(runner.executor, executor);
         assert.strictEqual(runner.modelPath, modelPath);
         assert.strictEqual(runner.inputSpec, inputSpec);
       });
     });
 
     suite('#getInferenceCmd()', function() {
       test('gets inference command', function() {
-        let runner = new InferenceRunner(backend, modelPath, inputSpec);
+        let runner = new InferenceRunner(
+            backend, (backend.executors() as Executor[])[0], modelPath, inputSpec);
```

Comment:
Related to https://github.com/Samsung/ONE-vscode/pull/1094/files#r928556316,

(backend.executors() as Executor[])[0] looks a bit unnatural.

Reply:
It looks unnatural to test only the first executor.
How about running the inference command for all the registered executors from the code, for example?